src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1444,6 +1482,8 @@ components:
           required:
             - addresses
           properties:
+            opt_in_properties:
```

Comment:
OptIns vs OptInProperties... Somehow former looks better to me but I have no strong opinion on that.